### ISSUE TITLE
Upgrade torchvision

### ISSUE THREAD

**Issue by charleygros** [neuropoly/ivado-medical-imaging#210]
I thought we should put this question back on the table: Should we upgrade our torchvision?

@andreanne-lemay & @olix86 --> What are your thoughts?

Small new argument I can across today in favour of an upgrade --> This [new functio](https://pytorch.org/docs/stable/nn.functional.html#instance-norm)n `instance_norm` is not available in our current `torchvision` version but is exactly what we need for: [NormalizeInstance2D & 3D](https://github.com/neuropoly/ivado-medical-imaging/blob/master/ivadomed/transforms.py#L439).

**Comment by olix86** [comment]
I agree it would be nice to upgrade, but last time I checked we were being held back by our Nvidia driver version #69

**Comment by jcohenadad** [comment]
> I agree it would be nice to upgrade, but last time I checked we were being held back by our Nvidia driver version on rosenberg #69

could we consider updating our driver?

**Comment by olix86** [comment]
Yes that would be great for us, when we had the discussion in october we were blocked by the fact that ADS needed the old driver / CUDA for their version of TF, not sure if that's still true.
